### ISSUE TITLE
Check emails sending/blocked by BL

### ISSUE THREAD

**Issue by crisr15** [scientist-softserv/britishlibrary#228]
# Summary

BL is not receiving password resets (jenny's email).  And have had some issues with new account signup.  

We want to confirm that emails are sending.  And to see if the SMTP server knowns if they are getting blocked or not.  

# Acceptance Criteria
- [ ] Answer question definitively: Are emails sending
- [ ] Are BL emails getting blocked by firewall(etc) at BL? 
- [ ] If emails are getting blocked send information to BL and create a new ticket while working with BL Tech team.  (Probably something around giving IP address or account something to whitelist the emails). 

# Possible steps

We believe this is going through sendgrid, so looking in sendgrid to find bl and then look at message history may be a good first step.  Additionally, you can create and account and reset its password to see if emails are sending.

**Comment by j-basford** [comment]
tested 10:35GMT 2023-01-31 with a 'reset your password' email to my BL address and it came through fine. Sendgrid must be accepted at BL end.
